models/staging/hubspot_staging/stg_rag_hubspot__engagement_email.sql

```diff
@@ -44,7 +44,7 @@ final as (
         owner_id,
         team_id,
         coalesce(body_preview, body_preview_html, email_text, email_html) as body,
-        email_subject,
+        email_subject as title,
```

Comment:
This needs to be documented as a breaking change in the CHANGELOG

Reply:
added